Signal/src/environment/SignalApp.m

```diff
@@ -137,7 +137,7 @@ - (NotificationsManager *)notificationsManager
     return _notificationsManager;
 }
 
-- (AccountManager *)accountManager
+- (nonnull AccountManager *)accountManager
```

Comment:
Hmmm, why is this necessary? 

Looks like the whole .m is within `NS_ASSUME_NONNULL_BEGIN`

Reply:
Ah, looks like this was a false positive (or dirty workspace). Will fix. 👌